### DISCUSSION TITLE
希望能增加下滑是复制的功能

### DISCUSSION THREAD

**Discussion by Nahuimi** [fcitx5-android/fcitx5-android#495] [Ideas | 想法]
目前上滑是能符号及数字，我希望同时下滑也有功能，例如c下滑是复制上滑是”，v上滑是？，下滑粘贴，b下滑是剪切板等功能

**Comment by usc** [comment]
也想有这个功能，安卓中，如果想快速粘贴，现在能知道的有：1）输入框长按然后选择粘贴，2）fcitx5打开工具栏打开文本编辑工具里有粘贴，都不是很快速
